src/shared/discover-image.c

```diff
@@ -58,11 +58,13 @@ static const char* const image_search_path[_IMAGE_CLASS_MAX] = {
                             "/usr/local/lib/portables\0"
                             "/usr/lib/portables\0",
 
+        /* Note that we don't allow storing extensions under /usr/lib, unlike with other image types. That's
```

Comment:
just write `/usr/` instead of `/usr/lib/`? After all we don't allow `/usr/local/lib/` either...